src/hops/models.py

```diff
@@ -1,13 +1,12 @@
 from app import db
-from werkzeug.security import generate_password_hash, check_password_hash
 
 
 class Hop(db.Model):
     __tablename__ = "hops"
 
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(32), unique=True)
-    alpha = db.Column(db.String)
+    alpha = db.Column(db.JSON)
     beta = db.Column(db.JSON)
```

Comment:
As I remember, you mentioned changing these to integers/floats. Is that true?

Reply:
Yeah, you are right. I haven't done it yet, but I will fix it.

Reply:
fixed